src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+public class Node<T> {
+
+    private Node prev;
+    private T element;
+    private Node next;
+    private boolean head;
+    private boolean tail;
+
+    public Node(Node prev, T element, Node next, boolean head, boolean tail) {
+        this.prev = prev;
+        this.element = element;
+        this.next = next;
+        this.head = head;
+        this.tail = tail;
+    }
+
+    public Node getPrev() {
+        return prev;
+    }
+
+    public void setPrev(Node prev) {
```

Comment:
The checklist advises not to use getters and setters in the `Node` class. Since `Node` is an inner class, you can access its private fields directly.